spec/outputs/http_spec.rb

```diff
@@ -126,6 +126,10 @@ def sinatra_run_wait(app, opts)
                          with(expected_method, url, anything).
                          and_call_original
       allow(subject).to receive(:log_failure).with(any_args)
+      allow(subject).to receive(:log_retryable_response).with(any_args)
+    end
+
+    context "handling too many connections" do
```

Comment:
I don't see the reason for this, is a reminder that it this isn't tested?

Reply:
Removing...

Reply:
It was added in a previous draft, this is tested elsewhere.